### PR TITLE
[Merged by Bors] - Add getting started guide

### PR DESCRIPTION
**Pull request by fhennig**
# Description

#329 

## Review Checklist

- [ ] Code contains useful comments
- [ ] CRD change approved (or not applicable)
- [ ] (Integration-)Test cases added (or not applicable)
- [ ] Documentation added (or not applicable)
- [ ] Changelog updated (or not applicable)
- [ ] Cargo.toml only contains references to git tags (not specific commits or branches)
- [ ] Helm chart can be installed and deployed operator works (or not applicable)

Once the review is done, comment `bors r+` (or `bors merge`) to merge. [Further information](https://bors.tech/documentation/getting-started/#reviewing-pull-requests)

### CODE REVIEW

File: docs/modules/getting_started/examples/code/getting_started.sh [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+# end::helm-install-operators[]
+;;
+"stackablectl")
+echo "installing Operators with stackablectl"
+# tag::stackablectl-install-operators[]
+stackablectl operator install opa=0.10.0-nightly
+# end::stackablectl-install-operators[]
+;;
+*)
+echo "Need to give 'helm' or 'stackablectl' as an argument for which installation method to use!"
```
Comment (maltesander) [inline docs/modules/getting_started/examples/code/getting_started.sh:35]:
```suggestion
echo "Need to provide 'helm' or 'stackablectl' as an argument for which installation method to use!"
```

File: docs/modules/getting_started/examples/code/getting_started.sh.j2 [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+# end::helm-install-operators[]
+;;
+"stackablectl")
+echo "installing Operators with stackablectl"
+# tag::stackablectl-install-operators[]
+stackablectl operator install opa={{ versions.opa }}
+# end::stackablectl-install-operators[]
+;;
+*)
+echo "Need to give 'helm' or 'stackablectl' as an argument for which installation method to use!"
```
Comment (maltesander) [inline docs/modules/getting_started/examples/code/getting_started.sh.j2:35]:
```suggestion
echo "Need to provide 'helm' or 'stackablectl' as an argument for which installation method to use!"
```

**Review by maltesander — APPROVED** [review]
LGTM. Just minor things.

**Comment by fhennig** [comment]
bors merge

**Pull request closed without merging**